### PR TITLE
Add Enphase Envoy scan interval description

### PR DESCRIPTION
**Pull request by catsmanac**
## Proposed change

Document Enphase Envoy integration scan interval and how to change it.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/enphase_envoy.markdown [hunk 1]
```diff
@@ -59,3 +59,6 @@ When the mode entity is set to standard, you can simply set the state of the rel
 
 ### Battery Level
 When the relay mode is set to battery level, the relays will turn on and off based on the remaining battery level of your Encharge batteries. Two number entities are available to control the cutoff and restore levels for the relays. When the battery level drops below the cutoff level, the relays will turn off. When the battery level rises above the restore level, the relays will turn back on.
+
+## Scan Interval
+The default scan interval is 60 seconds. To customize the scan interval use a [time pattern](/docs/automation/trigger/#time-pattern-trigger) triggered automation for service action using [homeassistant.update_entity](/integrations/homeassistant#service-homeassistantupdate_entity) targeting the envoy device. To scan the Envoy every 15 seconds use /15 in the seconds field of the time pattern trigger.
```
Comment (bdraco) [inline source/_integrations/enphase_envoy.markdown]: If we have a base sample automation, we should link to that instead.

If we don't have one, it would be good to add it in another PR, than update this PR to point to the newly added example once it merges.

Comment (frenck) [reply source/_integrations/enphase_envoy.markdown]: We have general documentation on this, please link that instead.

https://www.home-assistant.io/common-tasks/general/#defining-a-custom-polling-interval

**Review by frenck — CHANGES_REQUESTED** [review]
See above comments.

File: source/_integrations/enphase_envoy.markdown [hunk 2]
```diff
@@ -59,3 +59,6 @@ When the mode entity is set to standard, you can simply set the state of the rel
 
 ### Battery Level
 When the relay mode is set to battery level, the relays will turn on and off based on the remaining battery level of your Encharge batteries. Two number entities are available to control the cutoff and restore levels for the relays. When the battery level drops below the cutoff level, the relays will turn off. When the battery level rises above the restore level, the relays will turn back on.
+
+## Polling Interval
+The default polling interval is 60 seconds. To customize the polling interval refer to [Defining a custom polling interval](/common-tasks/general/#defining-a-custom-polling-interval). Specify the envoy device as target of the service using the `+ choose device` button. Updating the envoy will also update the related devices like the inverters , no need to split into separate entities or add all inverter devices. When using multiple Envoys, add each of them as target or create separate custom polling intervals as needed.
```
Comment (frenck) [inline source/_integrations/enphase_envoy.markdown]:
```suggestion
The default polling interval is 60 seconds. To customize the polling interval, refer to [defining a custom polling interval](/common-tasks/general/#defining-a-custom-polling-interval). Specify the envoy device as a target of the service using the `+ choose device` button. Updating the envoy will also update the related devices like the inverters; there is no need to split them into separate entities or add all inverter devices. When using multiple Envoys, add them as targets or create separate custom polling intervals as needed. 
```

**Review by frenck — APPROVED** [review]
Thanks, @catsmanac 👍 

../Frenck

**Comment by bdraco** [comment]
Thanks @catsmanac 

Thanks @frenck
